modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/network-observability-network-policy.adoc
+
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Create a network policy for Network Observability
```

Comment:
Do we want to stick with a gerund for this module title? eg "Creating" ? as per the guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings